.gitmodules

```diff
@@ -1,15 +1,15 @@
 [submodule "FV3"]
 	path = FV3
-	url = https://github.com/NOAA-EMC/fv3atm
-	branch = develop
+	url = https://github.com/climbfuji/fv3atm
+	branch = emc_devemc_update_from_dtc_develop_20191217
 [submodule "NEMS"]
 	path = NEMS
 	url = https://github.com/NOAA-EMC/NEMS
 	branch = develop
 [submodule "FMS"]
 	path = FMS
-	url = https://github.com/NOAA-EMC/FMS
-	branch = GFS-FMS
+	url = https://github.com/NOAA-GFDL/FMS
+	branch = master
```

Comment:
Is the master same as the tag: 2019.01? Currently the tag 2019.01 has revision 708b8d5, while master has revision:  cb85999

Reply:
Git does not checkout HEAD of the master, but commit specified explicitly in the repository tree with command 'git add FMS'.

Reply:
Yes, I was just typing this: The PR points to tag 2019.01 as required, not to the head of master. There is no way to specify in .gitmodules that a certain tag is used, only branches can be specified. The tag 2019.01 was cut from master (I looked through the git log history in FMS), hence using master (or deleting the branch = line entirely, because master is the default) is correct imo.